e2e/actions_test.go

```diff
@@ -8,70 +8,71 @@ import (
 
 	"github.com/ramendr/ramen/e2e/dractions"
 	"github.com/ramendr/ramen/e2e/testcontext"
+	"github.com/ramendr/ramen/e2e/util"
 )
 
 func DeployAction(t *testing.T) {
 	testCtx, err := testcontext.GetTestContext(t.Name())
 	if err != nil {
-		t.Error(err)
+		util.Fatal(t, "Failed to get test context", err)
 	}
 
 	if err := testCtx.Deployer.Deploy(testCtx.Workload); err != nil {
-		t.Error(err)
+		util.Fatal(t, "Deploy failed", err)
 	}
```

Comment:
Another way - we can use ["gotest.tools/v3/assert"](https://pkg.go.dev/gotest.tools/v3/assert) or used a modified version supporting our logger.

With this it can be:

```go
err = testCtx.Deployer.Deploy(testCtx.Workload)
assert.NilError(t, err, "Deploy failed")
```

Reply:
There is no good way to use a per-test logger with the assert package, so wrapping testing.T looks like a better way.